barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.hpp

```diff
@@ -150,16 +157,21 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
 
             auto output = FF(0);
             auto running_challenge = FF(1);
-            Utils::scale_and_batch_elements(relation_evaluations, alpha, running_challenge, output);
+
+            // Sum relation evaluations, batched by their corresponding relation separator challenge, to get the value
+            // of the full honk relation at a specific row
+            Utils::scale_and_batch_elements_without_linear_contributions(
```

Comment:
yeah, indeed, which is a very long name. now that I think about it again, it might be better to just overload `scale_and_batch_elements` which might be a bit more c++ idiomatic?